src/cbwd/main.cc

```diff
@@ -57,9 +62,25 @@ static void print_help() {
  * @param cfg The new configuration to set.
  */
 static void apply_new_configuration(configuration const& cfg) {
+  
+  auto console_sink = std::make_shared<spdlog::sinks::stdout_color_sink_mt>();
+  console_sink->set_level(spdlog::level::warn);
+  console_sink->set_pattern("[multi_sink_example] [%^%l%$] %v");
+
+  auto file_sink = std::make_shared<spdlog::sinks::basic_file_sink_mt>("logs/cbwd.txt", true);
+  file_sink->set_level(spdlog::level::trace);
```

Comment:
Those lines should be moved in the if condition below. And you must not use the name "logs/cbwd.txt" but the name given by the configuration: cfg.get_log_filename()